Few fixes

andrey-parkhomets opened the pull request:
- [x] - updated dependency `go-echarts`
- [x] - added `make dev` to use as  `make dev ARGS="apps-of-apps"`
- [x] - prefixed destination for ApplicationSet like `in-cluster` to avoid confusing  application destination`{{ .name }}` when it templated.

Syndlex commented on pkg/model/apps.go:
```diff
@@ -56,28 +57,33 @@ func (r *Apps) AddApps(item v1alpha1.Application) {
 		Name:      item.Name,
 		ManagedBy: tracking,
-		Cluster:   destName,
+		Cluster:   "in-cluster",
```
I don't know if this is working correctly when renaming `in-cluster`.
We have renamed our `in-cluster` because it was confusing for users.

https://github.com/argoproj/argo-cd/discussions/13219

andrey-parkhomets replied:
made it only for case when it templated to keep original expirience who do not have it templated. Please review.

andrey-parkhomets committed:
chore(appset): handle appset destination

When `item.Spec.Template.Spec.Destination.[Name|Server] templated
in a legend it looks ugly i.e. {{ .name }}, in such case
make it `in-cluster` to indicate that AppSet in argocd cluster.
